vcp-infra/.infra/common.yaml

```diff
@@ -37,7 +37,8 @@ stack:
           cpu: 2
           memory: 2Gi
       ingress:
-        oidcProtected: true
         paths:
           - path: /w/
             pathType: Prefix
+          - path: /cellxgene/
+            pathType: Prefix
```

Comment:
Adding `/cellxgene/` here ensures the proxy still routes autosave traffic when the explorer lives under multiple prefixes; without the extra path the new requests would 404.